scijava-ops-engine/src/test/java/org/scijava/ops/engine/matcher/impl/OpDescriptionTest.java

```diff
@@ -59,7 +59,7 @@ public void testOpClassDescription() {
 
 		String expected = "test.classDescription(\n\t " //
 			+ "Inputs:\n\t\tjava.lang.Double input1\n\t\tjava.lang.Double input2\n\t " //
-			+ "Outputs:\n\t\tjava.lang.Double output1\n)\n";
+			+ "Output:\n\t\tjava.lang.Double output1\n)\n";
 		String actual = info.toString();
 		Assertions.assertEquals(expected, actual);
 	}
```

Comment:
We may want to switch `info.toString()` to the current `Infos.describeMultiline` if we like that better